string mode
string linear_direction
string angular_direction
float64 target_x
float64 target_y
float64 target_theta
---
bool response
